packages/quantic/cypress/integration/sort/sort-expectations.ts

```diff
@@ -0,0 +1,48 @@
+import {InterceptAliases} from '../../page-objects/search';
+import {SearchExpectations} from '../search-expectations';
+import {SortSelector, SortSelectors} from './sort-selectors';
+
+function sortExpectations(selector: SortSelector) {
+  return {
+    selectedOption: (value: string, selected: boolean) => {
+      selector.selectedOption(value).should(selected ? 'exist' : 'not.exist');
+    },
+
+    displaySortDropdown: (display: boolean) => {
+      selector.listbox().should(display ? 'exist' : 'not.exist');
+    },
+
+    containsOptions: (values: string[]) => {
+      values.forEach((value) => {
+        selector.option(value).should('exist');
+      });
+    },
+
+    displayLocalizedLabel: (label: string) => {
```

Comment:
So far expectations starting with `display...` are used to validate whether elements are present. I would suggest to rename this expectation to `labelContains` instead.